### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by PerryBaran**

### CODE REVIEW

File: src/kata1.fizzBuzz.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+    return "FizzBuzz";
+  }
+  if (isDivisableBy3) {
+    return "Fizz";
+  }
+  if (isDivisableBy5) {
+    return "Buzz";
+  }
+  return number;
+};
```
Comment (PerryBaran) [inline src/kata1.fizzBuzz.js:14]: I would suggest leaving single line gaps between if statements to make it more readable

File: src/kata2.booleanToWord.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+  if (boolean === true) {
```
Comment (PerryBaran) [inline src/kata2.booleanToWord.js:2]: can be simplified as so `if(boolean){...}`

File: src/kata2.booleanToWord.js [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+    return "Yes";
+  }
+  if (boolean === false) {
+    return "No";
+  }
```
Comment (PerryBaran) [inline src/kata2.booleanToWord.js:7]: Can be further simplified using a ternary operator `return boolean ? "Yes" : "No"`

File: src/kata4.humanCatDogYears.js [hunk 4]
```diff
@@ -1,3 +1,11 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const AgeTwo = 24;
+  const remainingCatYears = (number - 2) * 4;
+  const catAge = remainingCatYears + AgeTwo;
+  const remainingDogYears = (number - 2) * 5;
+  const dogAge = remainingDogYears + AgeTwo;
+  const arr = [number, catAge, dogAge];
+  return arr;
+};
```
Comment (PerryBaran) [inline src/kata4.humanCatDogYears.js:9]: this passes the test cases, but if you test for human years = 1, you get `[1, 20, 19]` instead of the expected `[1, 15, 15]`

File: src/kata4.humanCatDogYears.js [hunk 5]
```diff
@@ -1,3 +1,11 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const AgeTwo = 24;
```
Comment (PerryBaran) [inline src/kata4.humanCatDogYears.js:2]: In Javascript, variables are usually written in camel case e.g. `ageTwo`
However, as this is a magic number, this can also be written using all caps snake case e.g. `AGE_TWO`

File: src/kata5.reachDestination.js [hunk 6]
```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+  const arrivalTime = distance / speed;
+  const timeRoundedUp = Math.round(arrivalTime * 2) / 2;
```
Comment (PerryBaran) [inline src/kata5.reachDestination.js:3]: `timeRoundedUp` variable name is confusing, as the the time is roundest to the nearest 0.5, which isn't necessarily rounded up, I would suggest something like `roundedTime` instead

File: src/kata5.reachDestination.js [hunk 7]
```diff
@@ -1,3 +1,6 @@
...
+  return `I should be there in ${timeRoundedUp} hours.`;
```
Comment (PerryBaran) [inline src/kata5.reachDestination.js:4]: small nitpick, but if `timeRoundedUp = 1` this returns the string `"I should be there in 1 hours"` rather than `"I should be there in 1 hour"`.

File: test/kata1.fizzBuzz.test.js [hunk 8]
```diff
@@ -1,11 +1,17 @@
...
+  it("returns Buzz when passed a multiple of 5", () => {
+    expect(fizzBuzz(5)).toBe("Buzz");
+  });
+  it("returns FizzBuzz when passed a multiple 3 and 5", () => {
+    expect(fizzBuzz(90)).toBe("FizzBuzz");
+  });
+  it("returns the number when it isn't a multiple of 3 or 5", () => {
+    expect(fizzBuzz(4)).toBe(4);
+    expect(fizzBuzz(11)).toBe(11);
+  });
```
Comment (PerryBaran) [inline test/kata1.fizzBuzz.test.js:16]: it's best practice to leave an empty line between tests to make the code more readable

File: test/kata4.humanCatDogYears.test.js [hunk 9]
```diff
@@ -1,3 +1,8 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  it("returns the age in human years, cat years and dog years", () => {
+    expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
+    expect(humanCatDogYears(20)).toEqual([20, 96, 114]);
```
Comment (PerryBaran) [inline test/kata4.humanCatDogYears.test.js:6]: as mentioned above, you should try test for all edge cases, I would add tests for if human years = 1 and human years = 2

File: test/kata6.joinNames.test.js [hunk 10]
```diff
@@ -1,5 +1,38 @@
...
+          name: "Bart",
+        },
+
+        {
+          name: "Lisa",
+        },
+
+        {
+          name: "Maggie",
+        },
```
Comment (PerryBaran) [inline test/kata6.joinNames.test.js:17]: remove empty lines between objects

File: test/kata6.joinNames.test.js [hunk 11]
```diff
@@ -1,5 +1,38 @@
...
+      ])
+    ).toBe("Bart, Lisa & Maggie");
+    expect(
+      joinNames([
+        {
+          name: "Blanche",
+        },
+
```
Comment (PerryBaran) [inline test/kata6.joinNames.test.js:25]: remove empty line

File: test/kata7.getEmployerRole.test.js [hunk 12]
```diff
@@ -1,5 +1,28 @@
...
+      {
+        name: "Jenny",
+ 
+        role: "Sales Associate",
+      },
+      {
+        name: "Javid",
+
+        role: "Human Recommended Reading Assistant",
+      },
```
Comment (PerryBaran) [inline test/kata7.getEmployerRole.test.js:20]: remove empty line between `name` and `role` in objects

File: src/kata2.booleanToWord.js [hunk 13]
```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+  if (boolean === true) {
+    return "Yes";
+  }
+  if (boolean === false) {
```
Comment (PerryBaran) [inline src/kata2.booleanToWord.js:5]: unnecessary if statement, if boolean is true that case is already covered in the first if statement, you can just return "No", e.g.
```
if (boolean) {
  return "Yes"
}
return "No"
```

File: src/kata3.numberToReversedDigits.js [hunk 14]
```diff
@@ -1,3 +1,5 @@
-const numberToReversedDigits = (number) => {};
-
+const numberToReversedDigits = (number) => {
+  const arr = Array.from(String(number), Number);
+  return arr.reverse();
+};
 module.exports = numberToReversedDigits;
```
Comment (PerryBaran) [inline src/kata3.numberToReversedDigits.js:5]: should be an empty line between the function and the module.exports

**Review by PerryBaran — COMMENTED** [review]
Really good all round. All test cases pass and your code is very readable.

Good Stuff:
- Really clear naming of variables
- consistent use of double quotes/ not mixing single and double quotes
- consistent indentation throughout

Could be improved:
- spacing out code with empty lines to make it more readable
- more tests to test for edge cases
